### ISSUE TITLE
are there any data on fishes?

### ISSUE THREAD

**Issue by Ivan-Da-Dev** [theBowja/genshin-db#67]
are there any data on fishes? such as their name, image etc

**Comment by theBowja** [comment]
Yes. `genshindb.materials('Golden Koi')`

Unfortunately the category 'Fish' overlaps with the material that is also called 'Fish'.

I'll fix this.

**Comment by theBowja** [comment]
fixed in v3.12.3

`genshindb.materials('Fish', { matchNames: false, matchCategories: true })`

```
[
  'Abiding Angelfish',      'Aizen Medaka',
  'Akai Maou',              'Betta',
  'Bitter Pufferfish',      'Brown Shirakodai',
  'Crystalfish',            'Dawncatcher',
  'Glaze Medaka',           'Golden Koi',
  'Lunged Stickleback',     'Medaka',
  'Pufferfish',             'Purple Shirakodai',
  'Raimei Angelfish',       'Rusty Koi',
  'Snowstrider',            'Sweet-Flower Medaka',
  'Tea-Colored Shirakodai', 'Venomspine Fish'
]
```

**Comment by theBowja** [comment]
I have data for fishing pool names, fish spawn frequency, pool descriptions, etc. as well.

Let me know if you need them and I can parse them out for you.
